### ISSUE TITLE
Add .nvmrc to configure Node Version Manager for this project

### ISSUE THREAD

**Issue by jkwuc89** [node-apn/node-apn#571]
[Node Version Manager](https://github.com/creationix/nvm) will use .nvmrc to set the correct version of Node.js for a project. I recommend the following inside .nvmrc to use the stable version of node.

```
stable
```

**Comment by florianreinhart** [comment]
The `.nvmrc` will only be used during development, right?

When and how will nvm select the specified node version? Is there a special command you need to run?

**Comment by jkwuc89** [comment]
With nvm installed, running `nvm use` will attempt to use .nvmrc to switch to the version of node for this project. I have this project cloned and have added .nvmrc as described above. Running `nvm use` results in the following:

```
[ Keith-Wedingers-MBP:~/projects/git/node-apn (master)* ] $ nvm use
Found '/Users/kwedinger/projects/git/node-apn/.nvmrc' with version <stable>
Now using node v7.9.0 (npm v4.2.0)
```

**Comment by florianreinhart** [comment]
Ok, sounds good. Can submit a PR?

**Comment by florianreinhart** [comment]
Fixed by #573
